Makefile

```diff
@@ -1,7 +1,9 @@
 -include ../../config.mk
 -include ../../tools/kconfig/.config
 
-CC:=$(CROSS_COMPILE)gcc
+OS:=$(shell uname)
+
+CC:=$(CROSS_COMPILE)clang
```

Comment:
I'd prefer to keep the default on GCC, otherwise the build would break for people who don't have clang installed.